src/apis/SearchApi.ts

```diff
@@ -0,0 +1,14 @@
+
+import axios from 'axios';
+import { SearchRequest, SearchResponse } from '../types/SearchTypes';
```

Comment:
We don't have file `../types/SearchTypes`. Please define `SearchRequest` and `SearchResponse ` locally